gateway/1.10/modules/ROOT/pages/flex-gateway-managed-set-up.adoc

```diff
@@ -1,18 +1,29 @@
-= Deploy a Managed Flex Gateway to CloudHub 2.0
+= Deploy a Managed Flex Gateway
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
 
-Deploy a Managed Flex Gateway to a CloudHub 2.0 private space to launch a completely cloud hosted gateway. Configure the Managed Flex Gateway as an ingress to you private space by exposing a public endpoint, or configure the gateway to only handle internal traffic. You can have one or more Managed Gateway in each environment that the private space is shared with. Gateways can protect any API from the same environment of the gateway or APIs in an external network.
+Deploy a Managed Flex Gateway to a CloudHub 2.0 private space or Runtime Fabric namespace to launch a completely managed gateway. Configure the Managed Flex Gateway as an ingress to your target space by exposing a public endpoint, or configure the gateway to only handle internal traffic. You can have one or more Managed Gateway in each environment that the target space is shared with. Gateways can protect any instance from the same environment of the gateway or services in an external network.
 
 == Before You Begin
 
 Before getting started with Flex Gateway, ensure:
 
 * The business group you want to deploy your gateway to has Managed Flex Gateway Resources. To redestribute Managed Flex Gateway resources to a business group, see xref:access-management::business-groups.adoc#redistribute-resources[Redistribute Resources Between Existing Business Groups].
-* You have created a private space in CloudHub 2.0 or have access to a private space created by someone else. To create a private space, see xref:cloudhub-2::ps-create-configure.adoc[].
++ 
+NOTE: Gateways cannot be shared across business groups. A user must be a member of the business group to access the Flex Gateway. 
+* You have created or have access to a deployment space:
+** xref:cloudhub-2::ps-create-configure.adoc[Create a CloudHub 2.0 Private Space]
```

Comment:
Future: consider removing the hard-coded link text and let antora fill it in. Less maintenance.